assets/js/vue/FileManager.vue

```diff
@@ -73,13 +97,86 @@ const deleteItem = (item) => {
                             resolve();
                         })
                 }).catch(error => {
-                    reject(error)})
+                    reject(error)
+                })
         } else {
             reject(new Error('Cannot delete folders'));
         }
     })
 }
 
+const uploadFileChunk = (fileData, uploadInfo, destinationDirectory) => {
+    destinationDir = destinationDirectory.path;
+    return new Promise((resolve, reject) => {
+        resolve();
```

Comment:
We might be able to use this function instead of the dropZone drop area. But for now we'll move on to test.